eis_toolkit/feature_importance/feature_importance.py

```diff
@@ -0,0 +1,48 @@
+import numpy as np
+import pandas
+import pandas as pd
+import sklearn.neural_network
+from sklearn.inspection import permutation_importance
+
+from eis_toolkit.exceptions import InvalidDatasetException
+
+
+def evaluate_feature_importance(
+    clf: sklearn.neural_network or sklearn.linear_model,
+    x_test: np.ndarray,
+    y_test: np.ndarray,
+    feature_names: list[str],
```

Comment:
Is any sequence ok here? We have used `Sequence[str]` for many cases before to allow tuples etc.